client/src/featureform/resources.py

```diff
@@ -1322,6 +1428,29 @@ def operation_type() -> OperationType:
     def type() -> str:
         return "training-set"
 
+    def get(self, stub):
+        name_variant = pb.NameVariant(name=self.name, variant=self.variant)
+        ts = None
+        for x in stub.GetTrainingSetVariants(iter([name_variant])):
+            ts = x
+            break
+
+        return TrainingSet(
+            name=ts.name,
+            variant=ts.variant,
+            owner=ts.owner,
+            description=ts.description,
+            status=ts.status.Status._enum_type.values[ts.status.status].name,
+            label=(ts.label.name, ts.label.variant),
+            features=[(f.name, f.variant) for f in ts.features],
+            feature_lags=[],
+            provider=ts.provider,
+            # TODO: apply values from proto
+            tags=[],
```

Comment:
Shouldn't we set tags and properties? Ok if not easy to do but lets make note of it so we actually do it

Reply:
we should. I can do it as a separate PR